pennylane/qchem/dipole.py

```diff
@@ -205,7 +206,7 @@ def _fermionic_dipole(*args):
         """
         constants, integrals = dipole_integrals(mol, core, active)(*args)
 
-        nd = [anp.array([0]), anp.array([0]), anp.array([0])]
+        nd = [np.array([0]), np.array([0]), np.array([0])]
         for i, s in enumerate(mol.symbols):  # nuclear contributions
             nd[0] = nd[0] + atomic_numbers[s] * mol.coordinates[i][0]
```

Comment:
What is the type of the molecule's coordinate?

Reply:
I think `type(mol.coordinates[i][j])` will be a `float`.

Reply:
Typically, `<class 'pennylane.numpy.tensor.tensor'>`

They are differentiable, so could be jax objects too.